### DISCUSSION TITLE
Using Pre-trained Models/Weights  for Neural Receiver for OFDM SIMO Systems

### DISCUSSION THREAD

**Discussion by yhuwhai** [NVlabs/sionna#224] [Q&A] [answered]
Hi everyone, 

I am new to Sionna .

 I would like to replicate the results for Neural Receiver for OFDM SIMO systems. 

Is there a way to access the pre-trained weights/model from the Jupyter notebook?

What I have tried doing is to run the codes again to save the weights, however the simulation has been running for days. I guess I am missing something.

I need help on accessing the pre-trained models to speed up the results replication. 

Thank you.

**Comment by jhoydis** [comment, marked as answer]
Hi,

We currently do not provide the pre-trained weights (contrary to what is said in the notebook). We'll look into if and how that could be enabled.

Concerning the simulation and training time, it should take a few hours on a modern GPU, but not definitely not days. What hardware are you using?

**Comment by yhuwhai** [comment]
Hi Jakob,

Thanks for the quick feedback.

I am using a laptop with a CPU (Intel Core i5 10th generation).


Best,
Uyoata

**Comment by saiksaketh** [comment]
I think recently a gdrive link was added with the weights, so this can be closed
